Purchases/Misc/RCSystemInfo.m

```diff
@@ -15,6 +15,7 @@ @interface RCSystemInfo()
 
 @property(nonatomic, copy, nullable) NSString *platformFlavor;
 @property(nonatomic, copy, nullable) NSString *platformFlavorVersion;
+@property(nonatomic, readwrite) BOOL autoSyncPurchases;
```

Comment:
I think it would make more sense to store the whole settings object instead, so we don't need to add one property for each new setting.